### PR TITLE
Overriding Tornado's hard limit for file uploads (for Java apps).

### PR DESCRIPTION
**Pull request by menivaitsi**
https://ocd.appscale.com:8080/job/Daily%20Build/701/

### CODE REVIEW

**Comment by menivaitsi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/705/

File: AppDB/blobstore/blobstore_server.py [hunk 1]
```diff
@@ -355,7 +356,9 @@ def main(port):
   """
   setup_env()
 
-  http_server = tornado.httpserver.HTTPServer(Application())
+  max_buffer_size = 2 * 1024 * 1024 * 1024  # 2GB
```
Comment (obino) [inline AppDB/blobstore/blobstore_server.py]: Can you make the 2GB a constant?

File: AppDB/blobstore/blobstore_server.py [hunk 2]
```diff
@@ -37,6 +37,7 @@
 
 from google.appengine.tools import dev_appserver_upload
 
+
```
Comment (obino) [inline AppDB/blobstore/blobstore_server.py]: Extra space here.

**Comment by obino** [comment]
:+1:

**Pull request merged**